py3-spello.yaml

```
@@ -0,0 +1,43 @@
+package:
+  name: py3-spello
+  version: 1.3.0.post0
```

it looks like the upstream project created a new release with `.post0` as a suffix, shortly after the 1.3.0 release.  To fix the failed CI checks we should be able to use the suggestions below, when 1.3.1 is released we will need to remove the `.post0` from the git-checkout tag config below

```suggestion
  version: 1.3.0
```